.ruby-version

```diff
@@ -1 +1 @@
-3.0.5
+3.3.0
```

Comment:
This should be the lowest ruby version supported by the gem (as per: https://docs.publishing.service.gov.uk/manual/publishing-a-ruby-gem.html#ruby-version-compatibility)

so if this gem still supports Ruby 3.0 then this should be 3.0.6. 

As we're adding 3.3 support we should probably drop 3.0 support and move the minimum version to be 3.1, which would mean this should be 3.1.4

Reply:
Sorry, my mistake!